libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,9 +204,28 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getIetfLocale = (ietfLocale) => {
-  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
-  return nonStandardLocaleMap[ietfLocale] || ietfLocale;
+const getUniversalNavLocale = () => {
+  const { locale: { prefix: configPrefix = '', ietf = '' } = {} } = getConfig() || {};
+  const prefix = configPrefix.replace('/', '');
+  const DEFAULTLANG = {
+    en_US: ['ae_en', 'africa', 'au', 'be_en', 'ca', 'cis_en', 'cy_en', 'eg_en', 'gr_en', 'hk_en', 'id_en', 'ie', 'il_en', 'in', 'kw_en', 'langstore', 'lu_en', 'mena_en', 'mt', 'my_en', 'ng', 'nz', 'ph_en', 'qa_en', 'sa_en', 'sg', 'th_en', 'uk', 'vn_en', 'za'],
+    fr_FR: ['be_fr', 'ca_fr', 'ch_fr', 'lu_fr'],
+    de_DE: ['at', 'ch_de', 'lu_de'],
+    es_ES: ['ar', 'ar_es', 'cl', 'co', 'cr', 'ec', 'gt', 'la', 'mx', 'pe', 'pr'],
+  };
+  const CUSTOMLANG = {
+    be_nl: 'nl_NL',
+    ch_it: 'it_IT',
+    hk_zh: 'zh_TW',
+    no: 'nb_NO',
+    cis_ru: 'ru_RU',
+  };
+  const prefixParts = prefix.split('_').reverse();
+
+  return Object.keys(DEFAULTLANG).find((key) => DEFAULTLANG[key].includes(prefix))
+         || CUSTOMLANG[prefix]
+         || (ietf.includes('-') ? ietf.replace('-', '_') : `${prefixParts[0].toLowerCase()}_${prefixParts[1].toUpperCase()}`)
```

Comment:
Are we certain their are no prefixes without a separator? Such as `africa`, `sea`, etc.? In that case we'd have just one prefix part so we might get unexpected results.

Reply:
`africa` is mapped to `en`, but for `sea` it would be a problem

Reply:
We don't have mapping for `sea`, but I added a fix to handle that situation 👍